dolphinscheduler-server/src/main/resources/master.properties

```
@@ -39,6 +39,9 @@
 # master commit task interval, the unit is millisecond
 #master.task.commit.interval=1000
 
+# master cache process definition
+#master.cache.process.definition=true
```

Should we turn on cache by default
```suggestion
master.cache.process.definition=true
```

---

we have already set the default value in the class MasterConfig.

---

For this situation, I think we better add comment in L42 to desc `# master cache process definition, default true` make it more clearer

---

you are right.
thank you for your suggestion.